RELEASE_NOTES

```diff
@@ -3,9 +3,42 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v2.8.2 (July 12th 2016)
+-----------------------
+
+bugfix release
+- various small enhancements, including:
+    - add support for rst output for --list-* and --avail-* (#1339)
+    - add support for 'eb --check-conflicts' (#1747, #1807, #1833)
+    - ensure nice error message when non-existing path is passed to apply_regex_substitutions (#1788)
```

Comment:
could you explain better what is `apply_regex_substitutions` about? (or, feed a pointer)

fi. if it was regex substitutions for build commands, it might have helped in a performance analysis challenge I've been touching today...